state_synchronizer/src/coordinator.rs

```diff
@@ -414,6 +420,8 @@ impl<T: ExecutorProxyTrait> SyncCoordinator<T> {
                 let mut req = GetChunkRequest::new();
                 req.set_known_version(self.known_version + offset);
                 req.set_limit(self.config.chunk_limit);
+                self.peer_manager
+                    .process_request(self.known_version + offset, peer_id);
```

Comment:
not sure, but should it be `self.known_version + offset + 1` . Can you double check?

Reply:
Yeah you're right, need the +1, didn't realize offset is always 0. Why is offset always zero?